templates/r10k-cache-pvc.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
```

Comment:
Do we really need another PVC? Can't we use an existing one?

Reply:
We probably could use an existing PVC.  We will just have to change the r10k config to use a different path for cache.  I will look into that.

Reply:
Yes, will be nice because most Cloud Providers does not have block storage less than 1 GB in size - so with every new PVC we add 1 GB storage pressure and increase the costs.